src/Renderer/LayeredMaterial.js

```diff
@@ -172,14 +171,6 @@ LayeredMaterial.prototype.dispose = function dispose() {
             }
         }
     }
-
-    var jT = new JavaTools();
-
-    jT.freeArray(this.textures[0]);
```

Comment:
No it's not needed.
One word was missing in @autra sentence: "The proper way to **explicitely** deallocate ...".
Even without setting these to `null` the GC will realize they are unreachable and thus candidate for deallocation.